dali/test/python/test_operator_numba_func.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+import os
+from numba import cfunc, types, carray
```

Comment:
I would add to `TL0_python-self-test/test_nofw.sh` and `TL1_python-self-test-slow/test.sh` numba as a requirement (even if it is implied by librosa installation)`. Just in case.